src/geometry/PolySetUtils.cc

```diff
@@ -90,15 +94,15 @@ void tessellate_faces(const PolySet& inps, PolySet& outps)
 
   // Estimate how many polygons we will need and preallocate.
   // This is usually an undercount, but still prevents a lot of reallocations.
-  outps.polygons.reserve(polygons.size() );
+  PolySetBuilder builder(verts.size(),polygons.size());
+  for(int i=0;i<verts.size();i++)
+    builder.vertexIndex({verts[i][0],verts[i][1],verts[i][2]});
```

Comment:
thinking about it, this function is using reindexer twice, once for removing degenerates and once for the builder.